src/core/server/base.js

```diff
@@ -160,55 +197,25 @@ function baseServer(routes, createStore, { appInstanceName = appName } = {}) {
     ) => {
       cookie.plugToRequest(req, res);
 
-      if (err) {
```

Comment:
Yeah, sorry, there isn't test coverage for the server code so I did a fair bit of manual stuff but haven't written many tests for it yet. I'll add that back in though as after I refactored the code it's easy to do. Thanks for catching it.